### ISSUE TITLE
Post Migration Issue

### ISSUE THREAD

**Issue by headcomic** [luetkemj/mcdc#37]
Kevin was going through the posts on the dev site because not all of them were there/there in the way they were supposed to be. I also know this happened to the last 10 gutter talks as well (that they just aren't there).

Message from Kevin:

"hey amigo
so im going through these drafts that are on the website
some of them are published...
and they have comments on them
that aren't showing up on the dashboard, but are showing up on the page itself
make a list so you can check em out
OH
I JUST DISCOVERED
that they didn't copy over correctly
from the site that's live right now
to the RD dashboard
they're on the dashboard on the normal making comics site
ok here's the list of them
It's Called Freelancing - Part 3
Don't fear the Timer
Photoshop Batch Actions for Comics
Drawing Pains
The Joy of Risk
Putting the “Com” in Comics
The Webcomic Graveyard
I've Been Framed!
Webcomic Success Through Advertising
How does one determine page rates when first starting out as a comic book artist?
And that's all of em
Basically every one of the articles labeled as drafts in the RD dashboard
now, i'd hit publish on them myself
but they're missing all the extra details
such as twitter quotes, featured images, and the comments that are on the originals
so i didn't want to publish em and fuck something up
but that's the buzz on my end"

**Comment by headcomic** [comment]
Are we doing one final migration of content before the site goes live? If so can we arrange a time for this so that I can make sure that all the articles are categorized on the new site correctly?
